src/k2/httpproxy/HTTPProxy.cpp

```diff
@@ -378,6 +378,31 @@ seastar::future<nlohmann::json> HTTPProxy::_handleWrite(nlohmann::json&& request
     });
 }
 
+seastar::future<std::tuple<k2::Status,dto::CreateSchemaResponse>> HTTPProxy::_handleCreateSchema(
+    dto::CreateSchemaRequest&& request) {
+    K2LOG_D(k2::log::httpproxy, "Received create schema request {}", request);
+    return _client.createSchema(std::move(request.collectionName), std::move(request.schema))
+        .then([] (CreateSchemaResult&& result) {
+            return seastar::make_ready_future<std::tuple<k2::Status,dto::CreateSchemaResponse>>(
+                // Sending empty response as CreateSchemaResult doesn't have any other field than Status
+                std::make_pair(result.status, dto::CreateSchemaResponse()));
+        });
+}
+
+seastar::future<std::tuple<k2::Status, dto::Schema>> HTTPProxy::_handleGetSchema(dto::GetSchemaRequest&& request) {
+    K2LOG_D(k2::log::httpproxy, "Received get schema request {}", request);
+    return _client.getSchema(std::move(request.collectionName), std::move(request.schemaName), request.schemaVersion)
+    .then([](k2::GetSchemaResult&& result) mutable {
+        if (!result.schema) {
+            return seastar::make_ready_future<std::tuple<k2::Status, dto::Schema>>(
```

Comment:
same comment about using RPCResponse here and next line. Make sure to std::move() the status.